### ISSUE TITLE
Can not edit Datetime/Timestamp column value in table's Data view  when connecting to Oracle DB

### ISSUE THREAD

**Issue by qngl** [dbeaver/dbeaver#11685]
#### System information: 
- OS: Windows 10 Pro v1903  18362.592
- DBeaver version 21.0.0 or 7.3.5
- Java version 1.8.0_221
- Installed with dbeaver-ce-21.0.0-x86_64-setup.exe

#### Connection specification:
- Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Production With the Partitioning, OLAP, Data Mining and Real Application Testing options
- Oracle JDBC driver 12.2.0.1.0
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
When editing a Date/Datetime/Timestamp file in the table data editor, user can not save changes with an error popup.

#### Steps to reproduce, if exist:
- Open a table with Date/Datetime/Timestamp column;
- Edit 1 row by changing the Date/Datetime/Timestamp column;
- Click Save then get the error, saying that: [1843] ORA-01843: not a valid month
- The reason is that Oracle can not run the generated SQL when user click Save:
```
UPDATE ABC x SET x.RECEIVE_TIME='2020-09-07 11:05:54.301' WHERE x.ID=1;
```
It should be:
```
UPDATE ABC x SET x.RECEIVE_TIME=TIMESTAMP'2020-09-07 11:05:54.301' WHERE x.ID=1;
```

**Comment by LonwoLonwo** [comment]
Hello @qngl 
Can you provide more details, please?
Some info from table? like on the picture? 
Because we already use TIMESTAMP before TIMESTAMP columns data. 

![2021-03-11 08_56_06-DBeaver Enterprise 21 0 0 - DATES_TEST](https://user-images.githubusercontent.com/45152336/110742698-e1f2b280-8247-11eb-8769-e616bf90c8b5.png)


We have another problem, what "timestamp" word before data is and before DATE columns https://github.com/dbeaver/dbeaver/issues/8673

**Comment by qngl** [comment]
If user does inline editing and clicks "Save", then gets the error. 
![2021-03-12 09_35_51-DBeaver 21 0 0 - NOTIFICATION](https://user-images.githubusercontent.com/26473377/110878627-8511fd80-8316-11eb-8e5d-8f3fa4589d04.png)

**Comment by LonwoLonwo** [comment]
And what type of data has "RECEIVE_TIME"?

**Comment by qngl** [comment]
TIMESTAMP

**Comment by LonwoLonwo** [comment]
Ok, I understand. 
DBeaver add TIMESTAMP mark in Generated UPDATE dialog, but not in the real UPDATE statement

![2021-04-21 15_22_16-DBeaver Enterprise 21 0 3 - DATES_TEST](https://user-images.githubusercontent.com/45152336/115553251-df0ec700-a2b5-11eb-8196-7474c0733253.png)

**Comment by LonwoLonwo** [comment]
Hello @qngl
Can you please check "Data formats" settings.

Disable "Disable date/format formatting" helped me.

![2021-06-09 13_23_41-DBeaver Enterprise 21 1 0 - DATES_TEST](https://user-images.githubusercontent.com/45152336/121347050-70ea9600-c92f-11eb-9636-56cf148dab68.png)

**Comment by LonwoLonwo** [comment]
Insert data/timestamp values for Oracle with option "Disable date/time formatting" in preferences fixed in DBeaver 21.1.1

**Comment by Matvey16** [comment]
Verified
